.github/ISSUE_TEMPLATE/feature_request.yaml

```
@@ -0,0 +1,61 @@
+name: Feature Request
+description:  Create a feature request for new features
+labels: ["Type: Feature Request", "Needs: Triage", "Needs: Lead"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        **IMPORTANT: Before posting, remove sensitive data, such as passwords, secret keys, session cookies, etc.**
+
+        Thanks for showing interest in adding new features to openlibrary! 👍
+
+  - type: textarea
+    attributes:
+      label: Problem Description
```

```suggestion
      label: Problem
```
Just say "Problem" to match the but report template